src/ledger/LedgerManagerImpl.cpp

```diff
@@ -954,6 +957,23 @@ LedgerManagerImpl::closeLedger(LedgerCloseData const& ledgerData)
     // step 4
     mApp.getBucketManager().forgetUnreferencedBuckets();
 
+    // Maybe sleep for parameterized amount of time in simulation mode
+    auto numOps = std::accumulate(txs.begin(), txs.end(), size_t(0),
+                                  [](size_t s, TransactionFramePtr const& v) {
+                                      return s + v->getOperations().size();
+                                  });
+    auto sleepFor = std::chrono::microseconds{
+        mApp.getConfig().ARTIFICIALLY_SIMULATE_OP_APPLY_FOR_TESTING * numOps};
```

Comment:
should probably not be called `ARTIFICIALLY_SIMULATE_OP_APPLY_FOR_TESTING ` but something like `OP_APPLY_SLEEP_TIME_FOR_TESTING `